cypress/e2e/backoffice/file-manager-attachment/file-list.cy.ts

```diff
@@ -0,0 +1,136 @@
+import { container } from '@utils';
+import { UserLoginScenario } from '@scenarios/backoffice';
+import { FileManagerAttachmentDynamicFixtures, FileManagerAttachmentStaticFixtures } from '@interfaces/backoffice';
+import {
+  FileManagerAttachmentListPage,
+  FileManagerAttachmentAddPage,
+  FileManagerAttachmentViewPage,
+  FileManagerAttachmentDeletePage,
+  FileManagerAttachmentDetachPage,
+  FileManagerAttachmentAttachPage,
+} from '@pages/backoffice';
+
+describeForSsp('File Manager Module - Files List', { tags: ['@backoffice', '@fileManager', '@ssp'] }, () => {
+  const userLoginScenario = container.get(UserLoginScenario);
+  const fileManagerAttachmentListPage = container.get(FileManagerAttachmentListPage);
+
+  let dynamicFixtures: FileManagerAttachmentDynamicFixtures;
+  let staticFixtures: FileManagerAttachmentStaticFixtures;
+
+  before((): void => {
+    ({ dynamicFixtures, staticFixtures } = Cypress.env());
+  });
+
+  beforeEach(() => {
+    userLoginScenario.execute({
+      username: dynamicFixtures.rootUser.username,
+      password: staticFixtures.defaultPassword,
+    });
+  });
+
+  it('should access the Files List page in Backoffice', () => {
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.verifyListPage();
+  });
+
+  it('should successfully detach file from entity', () => {
+    const fileManagerAttachmentDetachPage = container.get(FileManagerAttachmentDetachPage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickViewButton();
+
+    fileManagerAttachmentDetachPage.detachFile();
+    fileManagerAttachmentDetachPage.verifySuccessMessage();
+  });
+
+  it('should upload multiple files with size constraints', () => {
+    const fileManagerAttachmentAddPage = container.get(FileManagerAttachmentAddPage);
+
+    // Prepare test files that meet the constraints
+    const testFiles: { fileContent: any; fileName: string; mimeType: string; filePath: string }[] = [];
+
+    cy.fixture('suite/backoffice/file-manager-attachment/test-files/document1.pdf', 'binary')
```

Comment:
probably this should be in page class and accept array of files?

Reply:
done